lib/iris/fileformats/grib/_save_rules.py

```diff
@@ -164,9 +164,16 @@ def identification(cube, grib):
     # operational product, operational test, research product, etc
     # (missing for now)
     gribapi.grib_set_long(grib, "productionStatusOfProcessedData", 255)
+
+    # Code table 1.4
     # analysis, forecast, processed satellite, processed radar,
-    # (analysis and forecast products for now)
-    gribapi.grib_set_long(grib, "typeOfProcessedData", 2)
+    if cube.coords('realization') and len(cube.coords('realization')) == 1:
```

Comment:
it's not checking for many realization coordinates, it's trying to look for one.
```
cube.coord('realization')
```
throws an exception if there are none, which is a bit messy in the code flow